pkg/apis/submariner.io/v1/types.go

```diff
@@ -77,6 +78,7 @@ type GatewayStatus struct {
 	Version       string       `json:"version"`
 	HAStatus      HAStatus     `json:"haStatus"`
 	LocalEndpoint EndpointSpec `json:"localEndpoint"`
+	Latency       string       `json:"latency"`
```

Comment:
May be we can include a composite field where we will report last/min/avg/max/mdev  fields? 

See the output of ping: 
```
3 packets transmitted, 3 received, 0% packet loss, time 2003ms
rtt min/avg/max/mdev = 2.552/2.647/2.837/0.134 ms
```

That gives a more meaningful picture of latency (instead of just instant latency)

Reply:
Changed it to a composite field.